src/components/App.tsx

```diff
@@ -4,23 +4,32 @@ import CssBaseline from '@mui/material/CssBaseline';
 import { ThemeProvider } from '@mui/material/styles';
 import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
 
+import store from '../common/store';
 import protocolTheme from '../theme/protocolTheme';
+import { fetchRecentRevisions } from '../thunks/searchThunk';
 import SearchView from './Search/SearchView';
 
-function App() {
-  return (
-    <React.Fragment key="App Component">
```

Comment:
while I'm here, I wonder why there is this key, it doesn't seem useful.
Also `React.Fragment` can be shortened to `<> .. </>`

Reply:
#99

unsure why I added the fragment but it isn't needed anymore